stubs/third_party_taint/lxml_sinks.pysa

```diff
@@ -1,2 +1,4 @@
-def lxml.etree.fromstring(input: TaintSink[XMLParser]): ...
+def lxml.etree.fromstring(input: TaintSink[XMLParser], parser, *, base_url): ...
```

Comment:
These extra parameters are actually not required by Pysa, and they make the model harder to maintain (more things to keep in sync with the stub), so we prefer to not include them.

(Apply throughout the diff)

```suggestion
def lxml.etree.fromstring(input: TaintSink[XMLParser]): ...
```